nixos/modules/module-list.nix

```
@@ -791,6 +791,8 @@
   ./services/network-filesystems/orangefs/client.nix
   ./services/network-filesystems/orangefs/server.nix
   ./services/network-filesystems/rsyncd.nix
+  ./services/network-filesystems/s3fs-fuse.nix
+  ./services/network-filesystems/samba.nix
   ./services/network-filesystems/samba-wsdd.nix
   ./services/network-filesystems/samba.nix
```

```suggestion
  ./services/network-filesystems/samba-wsdd.nix
  ./services/network-filesystems/samba.nix
```
You accidentlly added the samba.nix file a second time